### ISSUE TITLE
Visual adjustment megaticket

### ISSUE THREAD

**Issue by hobuobi** [crownshy/civic_os#142]
Documenting a few primarily text changes that will help with accessibility. Should be low-effort, high priority. 

## Open Poll

_Welcome Page_
- Remove the line underneath the title. 
- A little extra left-right padding on the carousel messages on mobile (1.5x whatever it currently is). 

_Voting_ 
- "Skip" -> "Unsure". (Also, should use DM Mono.) 
- Remove the dot next to "Someone says..." (At some point, we'll want to bring that back with a proper avatar and/or pseudonym, but it's just distracting for now)

_Contribute_
- Bold the first clause of each line in the instructions. 

_Demographics Page_
- (I'd like to increase the text size here, but the copy is too long right now.) 

_Ending Page_
- "GO DEEPER" should be in normal case ("Go Deeper") 

## Campaign 
- Header should be the exact same as on all other pages – place name capitalized and with the pin icon, and the sizing/dimensions of the action pill should match the others. All slightly off right now.
- Tighten up the heading – should be less space between "What should we do about" and the rest of the title, and the line-spacing for "AI and the future of our communities" should be tighter. Check Figma for exact specs. 
- Center the pills (Our Views > Listen & Learn > Decisions) 
- Content in the cards should be at 14pt (or whatever our usual body text size is). 
- Content in "What is this about" should be the same. 
- (Not just a text thing, but relevant) Remove the dots and lines from the cards. Can figure out how to reintegrate later, but it'll likely be a pain in the meantime to get right, and it takes up space from the content. 
- Content in the card buttons generally too long – adjustments: 
  - Our Views: "Join Now" 
  - Listen & Learn: "TBA" 
  - Decisions: "TBA"
- Pin icons in the bottom section aren't rendering. 
- hello@bloom-project.org should be hyperlinked (mailto:hello@bloom-project.org). Button should go to the same destination for now.

**Comment by dtodorowa** [comment]
Created sub-issues to better delegate work @rosswilson2306 those are up for grabs now

> Demographics Page: (I'd like to increase the text size here, but the copy is too long right now.)

Will wait for further instruction on this
